test/message-input.test.js

```diff
@@ -106,4 +106,16 @@ describe('message-input', () => {
       expect(textArea.inputElement.getAttribute('aria-label')).to.be.equal('Viesti');
     });
   });
+
+  describe('disabled attribute propagation', () => {
+    it('should be propagated to text-area', () => {
+      messageInput.disabled = true;
+      expect(textArea.getAttribute('disabled')).to.exist;
```

Comment:
```suggestion
      expect(textArea.disabled).to.be.true;
```

The `vaadin-text-area` has its own `disabled` property and that is the one we need to check.